.changeset/late-zebras-draw.md

```diff
@@ -0,0 +1,7 @@
+---
+'@openfn/language-hive': patch
+'@openfn/language-mysql': patch
+'@openfn/language-postgresql': patch
+---
+
+update `configuration-schema`
```

Comment:
Why are these release notes different? I prefer the more detailed notes in `tame-files-judge.md`

Reply:
Thanks for spotting this @josephjclark , the initial requirement changed a bit. 
I have added a changeset per adaptor change which covers the changes in more detail